src/components/Agreements/ViewAgreement/ViewAgreement.js

```diff
@@ -204,6 +233,22 @@ class ViewAgreement extends React.Component {
     return get(this.props.resources.invoices, ['records'], []);
   }
 
+  getOrganizations() {
+    const isPending = get(this.props.resources.interface, ['isPending'], true);
+    if (isPending) return undefined;
+    const interfaces = get(this.props.resources, ['interface', 'records', 0, 'interfaces'], []);
+    const interfaceMap = keyBy(interfaces, 'id');
+    const { orgs } = this.getAgreement();
+    const organizations = cloneDeep(orgs);
```

Comment:
Could we avoid this `cloneDeep` and just return the output from an `organizations.map()` instead of `organizations.forEach` instead?